### PR TITLE
#deploy_st2: deploy staging

### PR DESCRIPTION
**Pull request by ruslanpalagin**

### CODE REVIEW

File: contracts_arbitrum/hardhat.config.ts [hunk 1]
```diff
@@ -33,6 +33,8 @@ const hardhatNetwork = process.env.NO_FORK === 'true' ?
     tags: ["test", "local"]
   }
 
+console.log('process.env.URL', process.env.URL)
```
Comment (boivlad) [inline contracts_arbitrum/hardhat.config.ts:36]: may to rm console.log

**Review by boivlad — APPROVED** [review]

File: contracts_arbitrum/hardhat.config.ts (hunk 1 above)
Comment (boivlad) [reply contracts_arbitrum/hardhat.config.ts:36]: Do u deploy to dev env?

Comment (ruslanpalagin) [reply contracts_arbitrum/hardhat.config.ts:36]: I want to leave it as it is to make sure I use correct URL :)

Comment (ruslanpalagin) [reply contracts_arbitrum/hardhat.config.ts:36]: DEV: 
SCs - deployed
BE - not yet
FE - not yet
Plan: to deploy soon today

**Pull request merged**